src/library_manager/Kconfig

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Library manager configs
+
+menu "Loadable Modules"
+
+config LOADABLE_MODULES
+	bool "Loadable Modules Support"
+	default n
```

Comment:
does this depend on IPC_MAJOR_4?

Reply:
Yes. We need to add dependency in Kconfig.

Reply:
any update or ETA ?